index.html

```diff
@@ -0,0 +1,191 @@
+  				</div>
+          <div class="actions">
+            <a href="#" class="btn btn-lg">create free website</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="sfuture">
+      <div class="sfuture-container">
+        <div class="sfuture-content">
+          <div class="container">
+  					<h2 class="opacity-half">What holds the future?</h2>
+  					<p>AI, Copywriting and other services on demand.. You can decide.</p>
+  				</div>
+        </div>
+      </div>
+    </div>
+    <footer class="footer">
+      <div class="container">
+        <div class="footer-1">
+          <div class="row pad-0">
+            <div class="xs-12 md-6 col pad-l-0">
+              <h2 class="uppercase opacity-half letter-spacing-medium font-size-min mar-b-1">STAY IN TOUCH</h2>
+              <div class="subscribe">
+                <h3 class="font-weight-light font-size-md mar-b-1">Subscribe to Newsletter</h3>
+                <form action="#">
+                  <div class="width-100 cols jc-sb ai-c">
+                    <input class="bg-transparent width-70" type="email" name="email" placeholder="Your email" required="">
+                    <button type="submit" class="btn bg-white small-to-lg">
+                      <span class="btn-wrap">Subscribe</span>
+                    </button>
+                  </div>
+
+                  <p class="font-size-sm font-weight-normal opacity-half mar-b-2">You agree to receive emails with latest news. <a href="#">You can unsubscribe at any time.</a></p>
+                  <div class="notification margin-bottom-text overflow-hidden is-hidden"></div>
+                </form>
+              </div>
+              <div class="social-icons">
+
+              </div>
+            </div>
+            <div class="xs-12 md-6 col pad-r-0">
+              <h2 class="uppercase opacity-half letter-spacing-medium font-size-min mar-b-1">USEFUL LINKS</h2>
+              <div class="row pad-0 mar-0">
+                <div class="md-6">
+                  <h3 class="font-weight-light font-size-md mar-b-1">Product</h3>
+                  <ul>
+                    <li>Blog</li>
+                    <li>Exemples</li>
+                    <li>Pricing</li>
+                    <li>Support</li>
+                  </ul>
+                </div>
+                <div class="md-6">
+                  <h3 class="font-weight-light font-size-md mar-b-1">Company</h3>
+                  <ul>
+                    <li>Terms & Conditions</li>
+                    <li>Privacy Policy</li>
+                    <li>Contact</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="footer-2">
+          <!-- <div class="container"> -->
+            <div class="cols jc-sb ai-c">
+              <div>
+                <a href="#">
+                  <i class="icon heart"></i>
+                  <svg class="logo-svg" viewBox="0 0 103 29">
```

Comment:
This svg path is not needed. You have it on line 15 already.

Reply:
As it's present both on header and footer, is it possible to replicate it without duplicate the code ?